tests/functional/setup.yml

```diff
@@ -22,3 +22,16 @@
         state: present
       when:
         - not is_atomic
+
+    # why sleep 2? see here: https://github.com/ansible/ansible/issues/14413
+    - name: reboot the machine
```

Comment:
Interesting, I would have thought this would have closed the connection and the playbook would stop. We handled this with ``vagrant reload`` for the ``ceph-volume`` testing: https://github.com/ceph/ceph/blob/master/src/ceph-volume/ceph_volume/tests/functional/tox.ini#L47

To allow nodes using virtualbox to reboot correctly we did have to modify the Vagrantfile slightly though: https://github.com/ceph/ceph/blob/master/src/ceph-volume/ceph_volume/tests/functional/Vagrantfile#L338